scripts/ci_install.bash

```diff
@@ -0,0 +1,115 @@
+#!/bin/bash
```

Comment:
Is this the version we want in the master? I thought that already went in in another review.  How do we untangle this from CPER?